fedimint-core/src/util/mod.rs

```diff
@@ -105,8 +110,29 @@ impl SafeUrl {
         self.0.port()
     }
     pub fn port_or_known_default(&self) -> Option<u16> {
-        self.0.port_or_known_default()
+        if let Some(port) = self.port() {
+            return Some(port);
+        }
+        match self.0.scheme() {
+            // p2p port scheme
+            "fedimint" => Some(8173),
```

Comment:
should be some constant somewhere?

Reply:
IMO, it's like creating a constant for `"https`" or `443`.

These lazy slobs didn't do it: https://github.com/servo/rust-url/blob/de947abf896f44efc5f0fe382064fd95363dff37/url/src/parser.rs#L175 , so I think I think we don't have to either. :D

Reply:
I added the constant, but it wasn't easy to use it in the clap, or any other places, so I kind of left it there.